src/ducks/balance/Balance.jsx

```diff
@@ -123,19 +128,21 @@ class Balance extends PureComponent {
     const {
       accounts: accountsCollection,
       groups: groupsCollection,
-      settings: settingsCollection
+      settings: settingsCollection,
+      triggers: triggersCollection
     } = this.props
 
     if (isCollectionLoading(settingsCollection)) {
       return null
     }
 
     const settings = getDefaultedSettingsFromCollection(settingsCollection)
-
-    if (
-      isCollectionLoading(accountsCollection) ||
-      isCollectionLoading(groupsCollection)
-    ) {
+    const collections = [
+      accountsCollection,
+      groupsCollection,
+      triggersCollection
+    ]
+    if (some(collections, isCollectionLoading)) {
```

Comment:
Nit: I would use [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here as it's the same thing, but native:

```suggestion
    if (collections.some(isCollectionLoading)) {
```